content/_secrets.md

```diff
@@ -18,6 +18,7 @@ Attribute | Type | Description
 `updated_at` | datetime | The time the secret was last updated at.
 `value` | string | The secret's value.
 `visibility` | string | Defines who can see the secret. Can be private, tenant or global. **Supported for Cloudify Manager 4.3 and above.**
+`is_hidden_value` | boolean| Determines who can see the secret's value, if true the secret's value is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**
```

Comment:
@inbala Is this not optional like the other two?
`is_hidden_value` | boolean| Determines who can see the value of the secret. If True, the value of the secret is only shown to the user that created the secret and to admins. (default: false). **Supported for Cloudify Manager 4.4 and above.**

Reply:
It is, but in this section of attributes we just explain the attributes, the other parts describe it as a parameter to an API call.